### DISCUSSION TITLE
Setting a time for Sync on save

### DISCUSSION THREAD

**Discussion by casual-coderr** [remotely-save/remotely-save#646] [Ideas]
So currently there are two auto sync options.
1st is Schedule for Auto-run:
Where we keep a time of let's say 30 minutes such that the plugin syncs to the backend.
2nd is Sync on Save:
where every change is automatically synced to the backend.

In first method, most of the time, even when we are just reading and not doing any changes the plugin would run and it is unnecessary

In second method, the problem with obsidian is that it saves every time, even when a character is changed. 
So if i use either of the above options when using a s3 compitable service, there is a lot of write options happening. So I would request a feature of combining both of these or increasing the capability of the Sync on save: 

The idea is that after a change is registered, give the user a time like immediately (like now), 5,10,30,60 minutes before the sync plugin is operated. This would save a lot of unnecessary sync operations happening in the cloud.

**Comment by fyears** [comment]
this is kind of implemented. in method 2 the sync is not triggered on every keystroke, but it is triggered every x second.

**Reply by casual-coderr** [reply]
Is there currently a way to change this value of x second to suit my use case ?

**Reply by fyears** [reply]
no. it will make the settings really verbose
